flixel/FlxCamera.hx

```diff
@@ -555,13 +568,23 @@ class FlxCamera extends FlxBasic
 		{
 			_helperMatrix.identity();
 			_helperMatrix.translate(destPoint.x + frame.offset.x, destPoint.y + frame.offset.y);
-			var isColored:Bool = (cr != 1.0) || (cg != 1.0) || (cb != 1.0) || (ca != 1.0);
```

Comment:
Which check do you mean? Are you talking about the entire check now or just the alpha part?